api-object-annotation.go

```
@@ -0,0 +1,251 @@
+}
+
+// ObjectAnnotation describes a single annotation as returned by ListObjectAnnotations.
+type ObjectAnnotation struct {
+	Name         string
+	Size         int64
+	ETag         string
+	LastModified time.Time
+}
+
+// putObjectAnnotationOutput maps the PutObjectAnnotation XML response.
+type putObjectAnnotationOutput struct {
```

golangci-lint (2.12.2)

[error] type putObjectAnnotationOutput is unused

(unused)